index.js

```diff
@@ -80,8 +81,7 @@ function retryInit(options={}) {
 /**
  * Calculate the retry delay
  *
- * @param {Number} attempt Attempt count
- * @param {RetryOptions} options Retry options
+ * @param {RetryOptions} retryOptions Retry options
```

Comment:
Since it was also updated for `retryInit`:
```suggestion
 * @param {RetryOptions|Boolean} retryOptions Retry options
```